lib/voicebase/v2/response.rb

```diff
@@ -2,13 +2,40 @@ module VoiceBase
   module V2
     module Response
 
+      TRANSCRIPT_READY = "finished".freeze
+
       def success?
 
-        #todo the V2 API response does not include a "requestStatus" field
-        # need to determine Juergen's intent here.
+        # for the V1 API this indicates both a successful HTTP status code and a values of "SUCCESS" in the
+        # returned JSON. with V2, there is no "SUCCESS" value. The combined use was split, adding
+        # #transcript_ready? to both interfaces.
+
+        ok?
+      end
 
-        ok? #&& request_status == "SUCCESS"
+      def media_id
+        response['mediaId']
       end
+
+      def transcript_ready?
+        response['media']['status'].casecmp(TRANSCRIPT_READY) == 0
+      end
+
+      #todo double check the format for plain text transcriptions, but it's probably just text only
+
+      #todo need to split into JSON and plain text retrieval methods
+
+      def transcript
+        # this retrieves the JSON transcript only
+        response['media']['transcripts']['latest']['words']
```

Comment:
This looks a bit likely to run into `NoMethodError` errors on `nil` - Are you sure that entire hash syntax will be there 100% of the time?